src/renderer/app-bridge.ts

```diff
@@ -85,6 +86,7 @@ export class AppBridge {
         switch (method) {
             case apiCmds.getVersionInfo:
                 const versionInfo = await ssf.getVersionInfo();
+                alert( JSON.stringify( versionInfo ) );
```

Comment:
Why do we need this?

Reply:
No, that was a mistake, I meant to remove it :) It is gone now.